pkg/manager/manager.go

```diff
@@ -523,11 +540,22 @@ func (s *FraktiManager) ListImages(ctx context.Context, req *kubeapi.ListImagesR
 func (s *FraktiManager) ImageStatus(ctx context.Context, req *kubeapi.ImageStatusRequest) (*kubeapi.ImageStatusResponse, error) {
 	glog.V(3).Infof("ImageStatus with request %s", req.String())
 
-	// NOTE: we only show image status of hyper runtime and assume privileged runtime image are the same
-	status, err := s.hyperImageService.ImageStatus(req.Image)
-	if err != nil {
-		glog.Errorf("ImageStatus from hyper image service failed: %v", err)
-		return nil, err
+	var (
+		status *kubeapi.Image
+		err    error
+	)
+	if s.unikernelImageService != nil && isUnikernelRuntimeImage(req.GetImage().GetImage()) {
```

Comment:
Why not create a separate `UnikernelManager` instead of hack existing `FraktiManager`? Then we can avoid these if else which looks ugly.

Reply:
Seems create a seprate `UnikernelManager` won't benifit this, all CRI request still get into `FraktiManager` first, we still need to route which request send to which runtime, or I cloud try to pull all image from three runtime?

Reply:
I mean, you don't have to use `FraktiManager` at all, since we have no requirement to run unikernel workloads togerther with Hyper or Docker on same node. So you can just start `UnikernelManager` instead of `FraktiManager` if unkernel runtime is enable.

@feiskyer Is it make sense to you? Or keep as it is? (It just begin come to me that unikernel users have no requirement for mixed runtime on same node, am I right?)

Reply:
One thing touches frakti manager is the IDs of container/sandbox/image. Frakti needs to known who they are belonging to. This could be done via an extra abstract layer, but for now, it still needs to change the FraktiManager.

> we have no requirement to run unikernel workloads togerther with Hyper or Docker on same node

This is not true. e.g. for clusters deployed via kubeadm, we are still running many services on hyper (kube-dns) and docker (apiserver, scheduler, controller manager ...).

Reply:
@feiskyer I'm referring to dedicated nodes for unikenel, which can be controlled by taint/tolerant. I am feeling if users want to try unikernel, they will focus only on unikernel, it's kind of different from container users.

Of course, current mode also works except making `FraktiManger` ugly and fat. We may need trade-off here.